packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -18851,6 +18932,60 @@ components:
         - purposeDescription
         - purposeIsFreeOfCharge
         - handlesPersonalData
+    RiskAnalysisFormTemplate:
+      type: object
+      additionalProperties: false
+      properties:
+        version:
+          type: string
+          minLength: 1
+          maxLength: 250
+        answers:
+          additionalProperties:
+            $ref: "#/components/schemas/RiskAnalysisTemplateAnswer"
+      required:
+        - version
+        - answers
+    RiskAnalysisTemplateAnswer:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          format: uuid
+        values:
+          type: array
+          items:
+            type: string
+            minLength: 1
+            maxLength: 2000
+        editable:
+          type: boolean
+        annotation:
+          $ref: "#/components/schemas/RiskAnalysisTemplateAnswerAnnotation"
```

Since we never use the annotation id in the new spec we defined, we could also just remove this sub schema and here expose it like this (the API converter will extract text from annotation obj and put it here)

```suggestion
        annotationText:
          type: string
```

---

Updated: [1538cd3](https://github.com/pagopa/interop-be-monorepo/pull/2587/commits/1538cd37d2035a575462dd01eb4216153e541d9a)